Libraries/test/TestFunctionFSharp/TestFunctionCSharp/TestFunctionCSharp.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="FSharp.Core" Version="*" />
+   <!-- <PackageReference Include="FSharp.Core" Version="*" /> -->

```

Comment:
nit: Is this meant to be removed? Just checking if it was from local testing and not meant to be committed?

Reply:
Yes and removed now. It was constantly causing versioning problems this reference and the reference brought in by the `TestFunctionFSharp` dependency project. Found out I could remove the whole think in this project so it would pick up the reference from `TestFunctionFSharp` and no more versioning pain. It seemed to break in the past every time there was a new version of .NET.